src/components/ActNowCard/enums.js

```diff
@@ -0,0 +1,9 @@
+const selectedCard = {
+  CONNECT: "connect",
+  FOLLOW: "follow",
+  PETITION: "petition",
+  SHARE: "share",
+  ACT: "act",
+};
+
+export default selectedCard;
```

Comment:
```suggestion
const Card = {
  CONNECT: "connect",
  FOLLOW: "follow",
  PETITION: "petition",
  SHARE: "share",
  ACT: "act",
};

export default card;
```

We can even call this file `cards.js` instead of generic `enums.js`.